orquestaconvert/workflows/base.py

```diff
@@ -30,6 +30,21 @@
     'workflow',
 ]
 
+UNSUPPORTED_MISTRAL_ACTIONS = [
+    'std.echo',
+    'std.email',
+    'std.javascript',
+    'std.js',
+    'std.ssh',
+]
+
+MISTRAL_ACTION_CONVERSION_TABLE = {
```

Comment:
I'm guessing the `http` action inputs don't match up 1-for-1?

https://docs.openstack.org/mistral/latest/user/wf_lang_v2.html#std-http
https://github.com/StackStorm/st2/blob/master/contrib/core/actions/http.yaml

Reply:
The [`http-request` action runner](https://docs.stackstorm.com/reference/runners.html#http-runner-http-request) covers additional options to `core.http`.

There are still two problematic options: `proxies` (`http_proxy` and `https_proxy` in ST2) and `verify` (`verify_ssl_cert` in ST2).

I'll dig into converting those as well.